--- mariadb-10.1.13/storage/tokudb/PerconaFT/buildheader/make_tdb.cc	2016-03-24 09:12:12.000000000 +0000
+++ mariadb-10.1.13-patched/storage/tokudb/PerconaFT/buildheader/make_tdb.cc	2016-04-01 17:14:49.612599726 +0000
@@ -581,7 +581,7 @@
 }
 
 
-int main (int argc, char *const argv[] __attribute__((__unused__))) {
+int main (int argc __attribute__((__unused__)), char *const argv[] __attribute__((__unused__))) {
     assert(argc==1);
 
     printf("#ifndef _DB_H\n");
